apps/challenges/aws_utils.py

```diff
@@ -1101,9 +1101,10 @@ def create_eks_nodegroup(challenge, cluster_name):
             amiType="AL2_x86_64_GPU",
             nodeRole=cluster_meta["EKS_NODEGROUP_ROLE_ARN"],
         )
+        logger.info("Nodegroup create: {}".format(response))
     except ClientError as e:
         logger.exception(e)
-        return response
+        return
```

Comment:
![image](https://user-images.githubusercontent.com/12206047/108745796-8ca96800-7509-11eb-8872-6d7ea6bf0814.png)
Can we please use these exceptions to handle it?

Reply:
ClientError handles it right? we are logging exception to sentry. The issue was we were accessing `response` variable in exception block where it is not initialized